sdk/eventhub/event-hubs/samples/sendEvents.ts

```diff
@@ -33,14 +33,14 @@ async function main(): Promise<void> {
 
   for (let index = 0; index < listOfScientists.length; index++) {
     const scientist = listOfScientists[index];
-    const data: EventData = {
+    const event = {
```

Comment:
eventData ?

Reply:
Accidentally removed EventData

Reply:
I actually meant the name of the constant i.e. change `event` to `eventData` ?

Reply:
Ohh ok